src/main/java/org/terasology/segmentedpaths/controllers/SegmentCacheSystem.java

```diff
@@ -29,13 +29,21 @@
 import java.util.HashMap;
 
 /**
- * Created by michaelpollind on 4/3/17.
+ * Cache system for segments. Constructs, stores and supplies segments based off their prefabs.
  */
 @RegisterSystem(RegisterMode.AUTHORITY)
 @Share(value = SegmentCacheSystem.class)
 public class SegmentCacheSystem extends BaseComponentSystem {
+
+    //A map that stores already constructed segments with their prefab's name as a key.
```

Comment:
Oh, just realized - this should be `as the key` or `as key`, and not `a key` - note that since key is unique, it should have the definite article (`the`) preceding it.